qa/suites/fs/bugs/conf

```diff
@@ -0,0 +1 @@
+.qa/cephfs/conf
```

Comment:
This sub-suite does not exist in Mimic. The QA test for https://github.com/ceph/ceph/pull/22139 was chosen not to be backported.

Reply:
@batrick Thanks; dropped.